install.sh

```diff
@@ -268,22 +261,20 @@ fi
 
 # Set the required OS based on inputs and installed distribution
 if [[ "$INSTALL_NAV" == "true" ]]; then
-	REQUIRED_OS_ID="Ubuntu"
-	REQUIRED_OS_CODENAME="trusty"
-	REQUIRED_OS_RELEASE="14.04"
-	ROS_VERSION="indigo"
-elif [[ "$(lsb_release -si)" == "Debian" ]]; then
-	REQUIRED_OS_ID="Debian"
-	REQUIRED_OS_CODENAME="jessie"
-	REQUIRED_OS_RELEASE="8.7"
-	ROS_VERSION="kinetic"
-else
-	REQUIRED_OS_ID="Ubuntu"
-	REQUIRED_OS_CODENAME="xenial"
-	REQUIRED_OS_RELEASE="16.04"
-	ROS_VERSION="kinetic"
+	if [[ "$(lsb_release -si)" == "Debian" ]]; then
```

Comment:
Watch your indentions buddy. This file uses tabs

Reply:
Aye, sorry about that, what I get for rushing this before class. From now on I'll use git diff to see what changes I'm making and review it versus what needs to be done. That's something I should have been doing from the beginning.

Reply:
Thanks for keeping my script pure @ironmig, #tabsforlife.